client/src/pages/MainTable.js

```diff
@@ -0,0 +1,93 @@
+import React, { useState, useEffect } from "react";
+
+const MainTable = () => {
+  console.log("Page Uploaded");
```

Comment:
We use `console.log` in development to help us code, but we don't usually commit it because we don't want to print these development statements on our production website. Can we remove all the `console.log`s in this file?

Reply:
Thanks already removed